README.md

```diff
@@ -241,6 +242,32 @@ export class OutboundSmsMessage implements OutboundMessage {
 }
 ```
 
+### <a name="CustomValidationToClassValidatorTranslation"></a> Custom Validation to class-vlidator translation
+
+Input:
+
+```java
+@Value
+public class Foo {
+    
+   @ComplexCustomValidation(length = 100)
+   private final String bar;
+}
+```
+
+Result:
+```typescript
+/* tslint:disable */
+/* eslint-disable */
+import { CommonValidationMessages } from './CommonValidationMessages';
+import { localize } from './Localization';
+
+export class Foo {
+   @ComplexValidator(100, { message: localize('must be valid element') })
```

Comment:
ne znam jesam ja sad nes krivo skuzio, al zas se drugacije zovu anotacija i dekorator (ComplexCustomValidation - ComplexValidator)

Reply:
da to je ok, to je feature kojeg imamo a to je da ono podržavamo da se dekorator i sama anotacija mogu drugačije zvati to se može konfati :)